stellar-core/debian/changelog

```diff
@@ -1,3 +1,7 @@
+stellar-core (_VERSION_-_BUILD_VERSION_) stable; urgency=medium
+
+  * added libtbb-dev Build-Depends
+
```

Comment:
I'm not sure if this will work. We mangle the the version in the build jobs so we may end up with duplicate which may break the build